### DISCUSSION TITLE
Images on Github pages are not displaying on web browsers (chrome, firefox, ME) but they are displaying perfectly on mobile

### DISCUSSION THREAD

**Discussion by gameleonpptgames** [community/community#22498] [Other Feature Feedback, Questions, & Ideas]
<p>Hello everyone!</p>
<p>This is the link to the GitHub website with all the pages. All the images are responding perfectly on all the pages except on the “games” page. And it is only on the web browsers not on mobile. <a href="https://github.com/gameleonpptgames" class="inline-onebox" rel="noopener nofollow ugc">gameleonpptgames · GitHub</a></p>
<p>I just created multiple pages for a website on Github pages and I’m having trouble displaying the images on the “games” page. I am confused because when I created this page the images where displaying perfectly for about a month and then suddenly they weren’t displaying anymore without me changing anything. The images are displaying and have not been effected on mobile.</p>
<p>I tried to restructure my folders and paths of the images but I am confused because it is working just fine on mobile. I checked my console and I am not getting any error messages.</p>
<p>I would appreciate any ideas of what I can do to fix this problem . Thank you so much!</p>

**Comment by jfullstackdev** [comment]
<p>yup there are no errors, and this is the appearance on web, when you hover, each tile will reveal the game</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/4/b4042bdca2197db761374aa1e239c1d575ee437e.jpeg" data-download-href="/uploads/short-url/pGuSFlTzOsP36opyl3kKemDTknA.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071488-8a326a97-960a-498d-b972-7f5ef63479d5.jpeg" alt="image" data-base62-sha1="pGuSFlTzOsP36opyl3kKemDTknA" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/4/b4042bdca2197db761374aa1e239c1d575ee437e_2_690x373.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/b/4/b4042bdca2197db761374aa1e239c1d575ee437e_2_1035x559.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/4/b4042bdca2197db761374aa1e239c1d575ee437e_2_1380x746.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/4/b4042bdca2197db761374aa1e239c1d575ee437e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 86.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>what is the supposed appearance on mobile ? can you have a screenshot ?</p>

**Comment by gameleonpptgames** [comment]
<p>Yes, this is a screenshot on mobile<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/e/fe6343da985b1d10856c4515fe77c8ef6f5c8162.jpeg" data-download-href="/uploads/short-url/AipX14v8Rcoe9nVoRdN7PbC7pgm.jpeg?dl=1" title="IMG_9215" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/98007819/181071489-60a7b5d6-1197-4e5d-9709-fb9f4189d8a9.jpeg" alt="IMG_9215" data-base62-sha1="AipX14v8Rcoe9nVoRdN7PbC7pgm" width="262" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe6343da985b1d10856c4515fe77c8ef6f5c8162_2_262x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe6343da985b1d10856c4515fe77c8ef6f5c8162_2_393x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe6343da985b1d10856c4515fe77c8ef6f5c8162_2_524x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe6343da985b1d10856c4515fe77c8ef6f5c8162_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">IMG_9215</span><span class="informations">1125×2144 119 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>does the mobile feature have hover feature too, like you need to tap it once ?  it might be the cause of the error,</p>
<p>try removing the hover feature and let’s see</p>

**Comment by gameleonpptgames** [comment]
<p>On the mobile feature you need to tap once but on the browser you just hover and the card flips. I tried your advice with removing the hover feature but it’s still acting up and not displaying the images</p>

**Comment by jfullstackdev** [comment]
<p>this is just a curios test, of course this is not correct, but it’s loading now, just to make sure it will load</p>
<p>I just remove the <code>&lt;div class="box"&gt;</code>, again this is just to test whether that images will be loaded</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/3/d3e8f5aab65f0dcdda7d67f50ad05dbeda1fb65d.jpeg" data-download-href="/uploads/short-url/ueDRjM5gFCVp9wxMJccclf3Fu2x.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071493-40279570-0cd8-4d1a-8ae1-3320d7fffead.jpeg" alt="image" data-base62-sha1="ueDRjM5gFCVp9wxMJccclf3Fu2x" width="556" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3e8f5aab65f0dcdda7d67f50ad05dbeda1fb65d_2_556x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3e8f5aab65f0dcdda7d67f50ad05dbeda1fb65d_2_834x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3e8f5aab65f0dcdda7d67f50ad05dbeda1fb65d_2_1112x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3e8f5aab65f0dcdda7d67f50ad05dbeda1fb65d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1158×1040 95 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>meaning, it has something to do with your divs</p>

**Comment by gameleonpptgames** [comment]
<p>Thanks for the advice. I worked on the divs in the container and I see what you mean when the div class =“box” is removed. I tried rearranging them because I thought that maybe it had something to do with specificity but I’ve been unable to get them to display. I also tried to add a z-index to the .container .box .body .imgContainer img attributes in the CSS to check if maybe the images were just behind the background but that did not work either. I’m just confused because the images are displaying on mobile and the code is the same other than a couple media queries for screen size adjustments.</p>

**Comment by jfullstackdev** [comment]
<p>yup, as for me, maybe I’ll be digging more to get the answer,</p>
<p>but yes, you are always free to post here any updates on that, we’ll help you on the process, at least for now, you already have the hint</p>

**Comment by gameleonpptgames** [comment]
<p>Yes, thank you.  I will continue to work on it and post any updates.</p>

**Comment by gameleonpptgames** [comment, marked as answer]
<p>Update: The images are displaying now. I was adding the z-index to the wrong attributes. The images were hiding behind the box. Thanks for the help!</p>
